src/main/kotlin/ai/digital/integration/server/deploy/internals/cluster/operator/OnPremHelper.kt

```diff
@@ -51,10 +51,13 @@ open class OnPremHelper(project: Project) : OperatorHelper(project) {
         val clusterName = onPremClusterName(name)
 
         project.logger.lifecycle("Operator is being undeployed")
-        undeployCis()
 
-        project.logger.lifecycle("PVCs are being deleted")
-        getKubectlHelper().deleteAllPVCs()
+        if (undeployCis()) {
```

Comment:
This piece of code is repeating 4 times.
Let's create a common function for that